chainreg/chainparams.go

```diff
@@ -50,6 +50,13 @@ var BitcoinSimNetParams = BitcoinNetParams{
 	CoinType: keychain.CoinTypeTestnet,
 }
 
+// BitcoinSigNetParams contains parameters specific to the signet test network.
+var BitcoinSigNetParams = BitcoinNetParams{
+	Params:   &bitcoinCfg.SigNetParams,
+	RPCPort:  "38334",
```

Comment:
Port should be 38332.

Reply:
Right. We didn't do the `if wallet then 38334 else 38332` dance in `btcd`. So had to add a special case in `lnd` since we always subtract 2 for `bitcoind`.